input/params.toml

```diff
@@ -42,8 +42,8 @@ offset_ref_initial_exp_growth_rate_prior_sd = 0.025
 autoreg_p_hosp_a = 1 # shape1 parameter of autoreg term on IHR(t) trend
 autoreg_p_hosp_b = 100 # shape2 parameter of autoreg term on IHR(t) trend
 eta_sd_sd = 0.01
-infection_feedback_prior_logmean = 6.37408 # log(mode) + q^2 mode = 500, q = 0.4
-infection_feedback_prior_logsd = 0.4
+infection_feedback_prior_logmean = 0 # log(mode) + q^2 mode = 1, q = 3
+infection_feedback_prior_logsd = 3
```

Comment:
I'm probably not understanding something, but is this is a median = 1 and `std = 8102.583912148207` prior replacing a  `median = 586.4456526813148` and `std = 264.62731766803165` prior. E.g standard parameterisation of Log-Normal?

Reply:
I.e. should we be interpreting this as a comparison between a confident high prior vs an uninformed prior?

Reply:
Yes and since the 90 day window probably doesn't let the model learn infection feedback much (particularly near the peak when its mostly been growing), I suspect we will need a stronger prior (e.g. more informative but lower, next thing @dylanhmorris and I discussed trying is `logmean = log(100)` , `logsd = log(3)` so `median = 100`, `stdev = 279`

Reply:
>  I discussed trying is `logmean = log(100)` , `logsd = log(3)` 

I think this agrees with my reasoning below on the $N \gamma$ prior and makes sense.